src/utils/index.js

```diff
@@ -58,3 +58,24 @@ export function isStale(resource, state, resourceIdField = 'uid') {
   const incomingVersion = parseInt(resource.metadata.resourceVersion, 10);
   return existingVersion > incomingVersion;
 }
+
+export function getGitValues(url) {
+  let copyUrl;
```

Comment:
This can be cleaned up a bit:

```js
let copyURL = url.toLowerCase().replace(/https?:\/\//, '');
```

Also, what's the reasoning behind including labels with empty string values if we can't extract repo info?

Reply:
I really didn't know what to do in that case... should I return an error if labels can't be extracted??

Reply:
if the labels can't be extracted we don't have a valid github URL, should we even get this far?

Reply:
Yeah, if the labels can't be extracted I just ended up returning a blank object & ultimately create the pipelineRun with just these two labels: `'tekton.dev/pipeline'` & `app`.